src/android/LocalNotification.java

```diff
@@ -290,6 +290,20 @@ private void update (JSONArray updates, CallbackContext command) {
 
         for (int i = 0; i < updates.length(); i++) {
             JSONObject update  = updates.optJSONObject(i);
+
+            if(!update.has("trigger")){
+                JSONObject trigger = new JSONObject();
+                try{
+                    trigger.put("type", "calendar");
+                    if(!update.get("every").equals("")){
+                        trigger.put("every", update.get("every"));
+                    }
+                    update.put("trigger", trigger);
+                }catch (JSONException e) {
+                    e.printStackTrace();
```

Comment:
You could add a log here with a message to be more clear what happened in case of failure